content/agent/_index.md

```diff
@@ -27,79 +36,168 @@ The Agent has three main parts: the collector, DogStatsD, and the forwarder:
 
 This is all controlled by one supervisor process. We keep this separate so you don't have to have the overhead of each application if you don't want to run all parts, although we generally recommend you do.
 
-<div class="alert alert-info">
-This documentation covers Agent versions 5.0.0 and above.
-</div>
 
-## Configuration management tools
+## What is the Agent v6?
 
-Manage the Datadog agent and [integrations](/integrations) using configuration management tools:
+Agent 6 is the latest major version of the Datadog Agent. The big difference between Agent 5 and Agent 6 is that Agent 6 is a complete rewrite of the core Agent in Golang. Golang has allowed us to take advantage of concurrency. In place of the three processes the Agent v5 used to run --*the Forwarder*, *the Collector*, and *DogStatsD*-- there is now only one process: the Agent. It also comes with a number of other core improvements:
 
-### Chef
-* [Chef Github project](https://github.com/DataDog/chef-datadog)
-* [Chef installation](https://app.datadoghq.com/account/settings#integrations/chef)
-* [Chef documentation](/integrations/chef)
+* Agent v6 has significantly improved resource usage over Agent v5:
+    
+    * It has decreased CPU usage:
+    * It has a decrease Memory usage:
+    * It uses fewer File Descriptors:
```

Comment:
should this be capitalized?

Reply:
Also: the memory line should probably be: `It uses less memory`